llpc/test/shaderdb/general/PipelineVsFs_TestBPermute64.pipe

```diff
@@ -0,0 +1,169 @@
+; NOTE: Assertions have been autogenerated by tool/update_llpc_test_checks.py UTC_ARGS: --check-pal-metadata
+; RUN: amdllpc -gfxip 11.0 -o - -filetype=asm %s | FileCheck -check-prefixes=CHECK %s
+
+[VsGlsl]
+#version 450
+#extension GL_KHR_shader_subgroup_shuffle : enable
+
+void main() {
+    uint index = gl_SubgroupInvocationID;
+    uint value = 4 * index;
+    uint data = subgroupShuffle(value, index ^ 8);
+
+    gl_Position = vec4(0.0, 0.0, data, 1.0);
+}
```

Comment:
This test has two weakness.

First, a shuffle by `gl_SubgroupInvocationID ^ 8` should really be implemented entirely differently, using DPP16 or v_permlane16_b32 (maybe one for your todo list?).

Second, there's a lot of noise in the resulting assembly. A better test would avoid that; which is slightly magical, but you can do it with an `.lgc` test like this:
```
define spir_func float @fn(float %value, i32 %index) !lgc.shaderstage !0 {
.entry:
  %r = call float (...) @lgc.create.subgroup.shuffle.f32(float %value, i32 %index)
  ret float %r
}

declare float @lgc.create.subgroup.shuffle.f32(...)

; ShaderStageCompute
!0 = !{i32 7}
```
The RUN line needs to invoke lgc instead of amdllpc.

Reply:
Is there a way to set the Subgroup size to 64 in this case?

The compute shader usually sets the waveSize to 64 (apparently there were some tests that confirmed a performance increase in that case). 
There are two ways to overwrite this: set the option `useSubgroupSize`. As far as I can tell, this happens in the `SPIRVReader.cpp`. I am not sure if this is called when calling lgc directly, though.
And the other option is by making the `workgroupSize` <= 32. In that case, the waveSize is also reduced to 32. 

Is there a way to make the workgroup > 32 here?